snorkel/contrib/fonduer/fonduer/parser.py

```diff
@@ -67,14 +70,14 @@ def __init__(self,
                  flatten_delim='',
                  lingual=True,                       # lingual
                  strip=True,
-                 replacements=[(u'[\u2010\u2011\u2012\u2013\u2014\u2212\uf02d]', '-')],
+                 replacements=[],#[(u'[\u2010\u2011\u2012\u2013\u2014\u2212\uf02d]', '-')],
```

Comment:
Will users be able to still easily look for dashes in their LFs, rather than needing to worry about all the different types (e.g. endash, emdash, subtract, etc.)? It's fine if we move that responsibility to the user, just curious about the motivation for the change.